fe2-android/app/src/test/java/com/github/dedis/popstellar/ui/pages/home/LaunchPageObject.java

```diff
@@ -0,0 +1,37 @@
+package com.github.dedis.popstellar.ui.pages.home;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+
+import androidx.annotation.IdRes;
+import androidx.test.espresso.ViewInteraction;
+
+import com.github.dedis.popstellar.R;
+
+public class LaunchPageObject {
+
+  @IdRes
+  public static int launchFragmentId() {
```

Comment:
it's hard to understand that this allow to get/set a property. It doesn't seem to launch anything. please try to use a noun instead of a verb, e.g. `launcherId` or `fragmentId`.

Reply:
`launchFragment` is because of the naming of the `LaunchFragment.java` . Moreover this line returns the id of the `LaunchFragment` and is used to test if it is displayed in tests.